app/views/posts/_expanded.html.erb

```diff
@@ -293,6 +293,33 @@
           </div>
         <% end %>
 
+        <% if is_question && post.children.length >= SiteSetting["TableOfContentsThreshold"] %>
```

Comment:
[**suggestion (non-blocking):**](https://conventionalcomments.org/) Add a third check here, which checks whether the Threshold is not equal to `-1` to allow disabling the ToC. Non-blocking because for now we could achieve the same effect with setting an unreasonably high answer count as threshold, but having an "official value" would be nicer of course.